tests/tests/test_provides_depends.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import subprocess
+import time
+
+from ..common_setup import enterprise_no_client
+from .common_update import update_image, common_update_procedure
+from .mendertesting import MenderTesting
+from ..MenderAPI import auth, auth_v2, deploy, image, logger
+
+from testutils.infra.device import MenderDevice
+
+
+class TestProvidesDependsEnterprise(MenderTesting):
```

Comment:
Hehehe. I was reviewing commit by commit so I added a comment asking for this test to be moved to its own file adding the Enterprise suffix... Good :+1: